extensions/podcasts/src/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { closeMainWindow } from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+
+export async function runAppleScriptAndCloseWindow(appleScript: string) {
+  await closeMainWindow();
+  await runAppleScript(appleScript);
+}
+
+export function buildPodcastScript(scriptToRunOnceActivated: string): string {
+  return `
+    set old to (path to frontmost application as text)
+    tell application "Podcasts"
+      reopen
+      activate
+    end tell
+    tell application "System Events" to tell process "Podcasts"
+      ${scriptToRunOnceActivated}
+      delay 0.1
+      set visible to true
+    end tell
+    activate application old
+`;
+}
+
+export async function togglePlayPause() {
+  const playScript = `key code 49 using {option down}`; // option + space
```

Comment:
On my machine, the toggle is just the space key...

Reply:
Hrm - thank you for catching this @fe9lix.  Ok - key codes may not work if it is different by keyboard layout or culture.

Does your menu look like:
![OnPaste 20220307-181409](https://user-images.githubusercontent.com/10678906/157127316-2113574a-832a-4c4b-b365-289455a43343.png)

Reply:
@fe9lix - can you confirm your menu looks the same as the image above (with different shortcut keys)?

Reply:
here's the menu on my machine:
<img width="203" alt="CleanShot 2022-03-09 at 09 05 45@2x" src="https://user-images.githubusercontent.com/213498/157398784-e0fd19ae-12ec-4559-ad91-3d48f7008cf9.png">

Reply:
@fe9lix - I have updated to using the menu items.

Is there a way to note in an extension it requires accessibility?